src/content/developers/docs/apis/json-rpc/index.md

```diff
@@ -0,0 +1,154 @@
+
+_[Jump ahead to endpoints](/developers/docs/apis/json-rpc/reference)_
+
+## Prerequisites {#prerequisites}
+
+Beyond being familiar with how API endpoints can be used to fetch JSON data, it might be helpful to understand the [Ethereum stack](/developers/docs/ethereum-stack/) and [Ethereum clients](/developers/docs/nodes-and-clients/).
+
+## Publish / Subscribe {#pub-sub}
+
+Publish / subscribe (pub/sub) is a method of using JSON-RPC notifications to subscribe to Ethereum events without needing to poll for them:
+
+- [Geth v1.4](https://geth.ethereum.org/docs/rpc/pubsub)
+
+- [Hyperledger Besu 1.3](https://besu.hyperledger.org/en/stable/HowTo/Interact/APIs/RPC-PubSub/)
+
+## JavaScript API {#javascript_api}
+
+To talk to an Ethereum node from inside a JavaScript application use the [web3.js](https://github.com/ChainSafe/web3.js) library, which gives a convenient interface for the RPC methods.
+
+See the [JavaScript API](/developers/docs/apis/javascript) for more details.
+
+## JSON-RPC Endpoint {#json-rpc_endpoints}
+
+Default JSON-RPC endpoints:
+
+| Client           |          URL          |
+| ---------------- | :-------------------: |
+| C++              | http://localhost:8545 |
+| Go               | http://localhost:8545 |
+| Py               | http://localhost:4000 |
+| Parity           | http://localhost:8545 |
+| Hyperledger Besu | http://localhost:8545 |
+
+### Go {#go}
+
+You can start the HTTP JSON-RPC with the `--rpc` flag:
+
+```bash
+geth --rpc
+```
+
+change the default port (8545) and listing address (localhost) with:
+
+```bash
+geth --rpc --rpcaddr <ip> --rpcport <portnumber>
+```
+
+If accessing the RPC from a browser, CORS will need to be enabled with the appropriate domain set. Otherwise, JavaScript calls are limit by the same-origin policy and requests will fail:
+
+```bash
+geth --rpc --rpccorsdomain "http://localhost:3000"
+```
+
+The JSON RPC can also be started from the [geth console](https://geth.ethereum.org/docs/interface/javascript-console) using the `admin.startRPC(addr, port)` command.
+
+### C++ {#cpp}
+
+You can start it by running `eth` application with `-j` option:
+
+```bash
+./eth -j
+```
+
+You can also specify JSON-RPC port (default is 8545):
+
+```bash
+./eth -j --json-rpc-port 8079
+```
+
+### Python {#python}
+
+In python the JSONRPC server is currently started by default and listens on `127.0.0.1:4000`
+
+You can change the port and listen address by giving a config option.
+
+`pyethapp -c jsonrpc.listen_port=4002 -c jsonrpc.listen_host=127.0.0.2 run`
+
+### Java {#java}
+
+Run a Besu node on mainnet with the HTTP JSON-RPC service enabled:
+
+```bash
+besu --rpc-http-enabled
+```
+
+More details can be found in the [documentation](https://besu.hyperledger.org/en/stable/Reference/CLI/CLI-Syntax/#rpc-http-enabled).
```

Comment:
Could we link to this https://besu.hyperledger.org/HowTo/Interact/APIs/Using-JSON-RPC-API/ which explains more how to run Besu with RPC and how to use it. The ref link is correct but we prefer to point to guides instead of refs directly. Also I suggest a clearer link name that will help for search and indexing that a generic one and include some examples of parameters you can find in this doc as they are described for other clients. Thanks!
```suggestion
Read the [Besu JSON-RPC documentation](https://besu.hyperledger.org/en/stable/HowTo/Interact/APIs/Using-JSON-RPC-API/) for details about configuring the service (listening ip, port, cors, ...).
```

Reply:
@NicolasMassart I'm in favor of this, thanks for posting. I agree this link is clearer, and the landing page is more inviting. 👍